temporal/build_id_versioning.go

```diff
@@ -39,9 +39,19 @@ const (
 	// version if possible. It may not be possible if the target task queue does not also have
 	// knowledge of the current worker's build ID.
 	// WARNING: Worker versioning is currently experimental
+	// Deprecated. This has the same effect as VersioningIntentInheritBuildID, use that instead.
 	VersioningIntentCompatible = internal.VersioningIntentCompatible
 	// VersioningIntentDefault indicates that the command should run on the target task queue's
 	// current overall-default build ID.
 	// WARNING: Worker versioning is currently experimental
+	// Deprecated. This has the same effect as VersioningIntentUseAssignmentRules, use that instead.
 	VersioningIntentDefault = internal.VersioningIntentDefault
+	// VersioningIntentInheritBuildID indicates the command should inherit the current Build ID of the
+	// Workflow triggering it, and not use Assignment Rules. (Redirect Rules are still applicable)
+	// This is the default behavior for commands running on the same Task Queue as the current worker.
+	VersioningIntentInheritBuildID= internal.VersioningIntentInheritBuildID
```

Comment:
```suggestion
	VersioningIntentInheritBuildID = internal.VersioningIntentInheritBuildID
```

Usually `go fmt` would catch this